master/buildbot/changes/p4poller.py

```diff
@@ -235,6 +235,16 @@ def _poll(self):
             args.extend(['-m', '1', '%s...' % (self.p4base,)])
 
         result = yield self._get_process_output(args)
+        # decode the result from its designated encoding
+        try:
+            result = bytes2unicode(result, self.encoding)
```

Comment:
I don't think so.  Elsewhere in this file, they use the same pattern of doing a decode and then catching a UnicodeError.